internal/interpreter/interpreter.go

```diff
@@ -175,6 +175,7 @@ func RunProgram(
 	program parser.Program,
 	vars map[string]string,
 	store Store,
+	featureFlags map[string]struct{},
```

Comment:
we pass the feature flags here directly (without further design patterns e.g. dependency injection). When a fflag will be available, we will parse it as a boolean field in the `programState` struct

this way
1. we can still control presence/absence of fflags in tests
2. the runtime will have access to passed feature flags in each moment